app/src/features/Joins/SqlInputJoinList.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState, useEffect, useCallback } from "react";
+
+import { Button, Grid, makeStyles, Typography } from "@material-ui/core";
+import AddIcon from "@material-ui/icons/AddCircleOutline";
+import { useTranslation } from "react-i18next";
+
+import Join from "features/Joins/Join";
+import {
+  useApiJoinsListQuery,
+  useApiJoinsDestroyMutation,
+} from "services/api/endpoints";
+import {
+  SQLInput,
+  Join as JoinType,
+} from "services/api/generated/api.generated";
+
+const useStyles = makeStyles(() => ({
+  button: {
+    textTransform: "none",
+  },
+}));
+
+type SqlInputJoinListProps = {
+  sqlInputId: SQLInput["id"];
+};
+
+const SqlInputJoinList = ({
```

Comment:
JoinList could be a generic component instead of having one for Filter, SqlInput etc

Reply:
you can make an issue if you want

Reply:
#646